docs: "What's new in Svelte" November newsletter (#881)
sveltejs/svelte.dev: apps/svelte.dev/content/blog/2024-12-01-whats-new-in-svelte-december-2024.md

---
title: "What's new in Svelte: November 2024"
description: 'New features in Svelte 5 and a ton of new community libraries'
author: Dani Sandoval
authorURL: https://dreamindani.com
---

I can't believe it's already the last newsletter of the year! So much has happened in the Svelte community - from two great Svelte Summits to the release Svelte 5 🎉

As we look forward to the next year, let's dive into what's new in the past month...

## What's new in Svelte

> The bugfix versions in 5.1.x addressed a bunch of issues reported by the community. So if you're noticing anything weird, make sure you upgrade to 5.1.17 or later and, as always, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG.md) for all the bug fixes.

Reviewer: ```suggestion
The bugfix versions in 5.1.x and 5.2.x addressed a bunch of issues reported by the community. So if you're noticing anything weird, make sure you upgrade to the latest version and, as always, check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/main/packages/svelte/CHANGELOG.md) for all the bug fixes.
```

The features since last month can be found below:

> - Static attributes are now inlined better in the compiled output (**5.2.0**, [#14269](https://github.com/sveltejs/svelte/pull/14269))

Reviewer: This was reverted
```suggestion
```

- [Jovian Moon's personal site](https://github.com/StephenGunn/jovian) features an interactive rocket ship to fly you straight to Jupiter (or anywhere else you want to go)
- [Font Preview](https://github.com/songkeys/font-preview) makes it easy to analyze and explore font characters in the browser
- [Dunks & Threes](https://dunksandthrees.com/) is a professional basketball analysis platform that provides data-driven NBA predictions
- [AdresseDuBien](https://adressedubien.com/) is a small utility to find addresses for real estate ads (for France) with a Libre Map implementation
- [PixelAttack](https://jason1610.github.io/PixelAttack/) takes an image, pixelizes it and makes the pixels fight each other
- [NeoHtop](https://github.com/Abdenasser/neohtop) is a modern, process monitoring tool that combines the power of terminal-based system monitors with a clean, desktop-native UI
- [WebVM](https://github.com/leaningtech/webvm) is a Linux virtual machine that runs in your browser
- [Sveltris](https://svelte.dev/playground/8594eea15c6244908c3efad284fe2aa4?version=5.2.1) is Tetris built entirely in the Svelte REPL

### Learning Resources

_Featuring Svelte Contributors and Ambassadors_

> - [Svelte 🧡](https://bsky.app/profile/ricciuti.me/feed/svelte-feed) is a Bluesky feed to keep up to date on Svelte news in that application

Reviewer 2: ```suggestion
- [Svelte 🧡](https://bsky.app/profile/paolo.ricciuti.me/feed/svelte-feed) is a Bluesky feed to keep up to date on Svelte news in that application
```

It should work the same but I changed my handle so better to have the right one

- [SvelteKit Streaming: The Complete Guide](https://khromov.se/sveltekit-streaming-the-complete-guide/) by Stanislav Khromov
- [Remote Workshop: Svelte 5 & Runes](https://ti.to/mainmatter/svelte-5-runes-feb-2025) is taking place February 6-7th, 2025 and taught by Paolo Ricciuti
- [Svelte 5: In Action | Build A Tasks App](https://www.youtube.com/watch?v=uSWMvDPpG0k) by Syntax

> - [Svelte Society London](https://www.youtube.com/watch?v=mxM208nCvHc) - featuring Paolo with his talk, *The joy of side projects ✨*

Reviewer 2: ```suggestion
- [Svelte Society London](https://www.youtube.com/watch?v=mxM208nCvHc) - featuring Paolo with his talk, *Herd your async tasks!*
```
Also in the same Livestream there's also the talk "Oh no SEO" from @spences10

- [Monicon](https://github.com/oktaysenkan/monicon) is an all-in-one icon library that provides 200,000+ icons from popular sets like Material Design, Feather, and Font Awesome

*Everything Else*
- [Svelte Multitone Image](https://stephane-vanraes.github.io/svelte-multitoneimage/) is a simple image renderer to apply multitone effects for Svelte
- [number-flow](https://github.com/barvian/number-flow) is a component to transition, format, and localize numbers
- [sveltednd](https://github.com/thisuxhq/SvelteDnD) is a lightweight, flexible drag and drop library for Svelte 5 applications
- [ShadEditor](https://github.com/Tsuzat/ShadEditor) is a highly extensible text editor for svelte made with TipTap and ShadCN UI
- [Tipex](https://www.npmjs.com/package/@friendofsvelte/tipex) is an advanced rich text editor based on Tiptap and Prosemirror
- [svelte-bundle](https://github.com/uhteddy/svelte-bundle) is a tool for bundling Svelte components into a single HTML file
- [svelte5-router](https://github.com/mateothegreat/svelte5-router) is an SPA router that allows you to divide & conquer your app with nested routers

> 

Reviewer 2: ```suggestion
- [@sheepdog/svelte](https://github.com/main matter/sheepdog) is a library to manage async tasks and concurrency with ease.

```

> - [@sheepdog/svelte](https://github.com/main matter/sheepdog) is a library to manage async tasks and concurrency with ease.

Reviewer 2: ```suggestion
- [@sheepdog/svelte](https://github.com/mainmatter/sheepdog) is a library to manage async tasks and concurrency with ease.
```

Ugh noticed this in the newsletter
